src/client/features/view/cy/events/click.js

```diff
@@ -9,6 +9,9 @@ const bindClick = (cy, callback) => {
       let name = data.label;
       let cy = evt.cy;
 
+      console.log(evt.target.scratch('_search-style-before'));
```

Comment:
Delete log statements

Reply:
Didn't realize that was still there. I have removed it in the last commit.